storage/addressing.go

```diff
@@ -67,6 +67,17 @@ func MergeRangeAddressing(left, merged *proto.RangeDescriptor) ([]client.Call, e
 	return calls, nil
 }
 
+// UpdateRangeAddressing overwrites the meta1 and meta2 range addressing
+// records for the descriptor.
+func UpdateRangeAddressing(desc *proto.RangeDescriptor) ([]client.Call, error) {
+	var calls []client.Call
+	var err error
+	if calls, err = updateRangeAddressing(calls, desc, putMeta); err != nil {
```

Comment:
This whole function could be replaced with `return updateRangeAddressing(calls, desc, putMeta)` (unless you're deliberately trying to ensure that one or the other of calls and err are nil, in which case there should be a comment to that effect).